MAINTAINERS.yml

```diff
@@ -2705,3 +2705,16 @@ Random:
     - include/zephyr/random/
   labels:
     - "area: Random"
+
+Testing with Renode:
+  status: maintained
+  maintainers:
+    - mateusz-holenko
```

Comment:
This user does not seem to be a contributor/collaborator on the Zephyr repository, [as required by the documentation](https://docs.zephyrproject.org/latest/project/project_roles.html#maintainer).

Reply:
He is a contributor though: https://github.com/zephyrproject-rtos/zephyr/commits?author=mateusz-holenko

Reply:
I just sent an invite to @mateusz-holenko, should be fine.

Reply:
Thanks @nashif, I accepted your invitation.

What is the next step to move forward?

Reply:
No objection to this from me.